tools/external_links_manager.py

```diff
@@ -0,0 +1,26 @@
+#  Drakkar-Software OctoBot
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
+import json
+import requests
+
+from config import GITHUB_RAW_CONTENT_URL, ASSETS_BRANCH, GITHUB_REPOSITORY, EXTERNAL_LINKS_FILE
+
+
+def get_external_link(link_key):
```

Comment:
I don't really see the interest of creating a new tool for this.

Reply:
It's not a specific feature for the web interface and can be reused for any server-side update without updating the client software.
Maybe another name would be more appropriate, something like online resource-manager ?
I'm thinking about possibilities like info messages to broadcast to all users for example

Reply:
What do you think of the correction I made ?

Reply:
Perfect 👍